src/safe_apps/models.py

```diff
@@ -1,7 +1,54 @@
+from enum import Enum
+
 from django.contrib.postgres.fields import ArrayField
+from django.core.validators import RegexValidator
 from django.db import models
 
 
+def _hostname_validator() -> RegexValidator:
+    """
+    Proudly taken from https://github.com/MehdioKhan/django-domainname
+    """
```

Comment:
This is licensed under GPL so it cannot be redistributed under an MIT license.

Isn't it possible to have simpler regex for this? I see that port numbers are also covered. Do we need those?

Reply:
> This is licensed under GPL so it cannot be redistributed under an MIT license.

Hmm, the code itself is a slightly modified URLValidator from Django with dropped additional checks 🤷‍♂️ 

> Isn't it possible to have simpler regex for this? I see that port numbers are also covered. Do we need those?

I believe it is possible, but from studying existing validators, I understood that it's not as simple as it seems, and I decided not to reinvent the wheel because it's guaranteed I'll miss some edge-case. Anyway, I've just pushed a simpler regex that seems to work for our problem.